From f6f3f89e723d26cdad16e0d70d21e2361b9ac8bb Mon Sep 17 00:00:00 2001
From: Noriko Hosoi <nhosoi@redhat.com>
Date: Thu, 28 Jul 2016 22:23:20 -0700
Subject: [PATCH 26/29] Ticket #48936 - Duplicate collation entries

Description: In the fix for "Ticket #53 - Need to update supported locales",
some locales were not set country and variant codes correctly, which caused
the duplicate matchintRules.  Also, ig-NG was mapped to a wrong locale.

An example of the broken matchingRules.
Before the fix>
  matchingRules: ( 2.16.840.1.113730.3.3.2.42.1 NAME 'caseIgnoreOrderingMatch-sk' DESC 'sk' SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
  matchingRules: ( 2.16.840.1.113730.3.3.2.211.1 NAME 'caseIgnoreOrderingMatch-sk' DESC 'sk' SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
After the fix>
  matchingRules: ( 2.16.840.1.113730.3.3.2.42.1 NAME 'caseIgnoreOrderingMatch-sk' DESC 'sk' SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )
  matchingRules: ( 2.16.840.1.113730.3.3.2.211.1 NAME 'caseIgnoreOrderingMatch-sk-SK' DESC 'sk-SK' SYNTAX 1.3.6.1.4.1.1466.115.121.1.15 )

https://fedorahosted.org/389/ticket/48936

Reviewed by rmeggins@redhat.com (Thank you, Rich!!)

(cherry picked from commit 3e3dff89c29afdf52a32e4d44f01bddedd60bcd3)
---
 ldap/schema/slapd-collations.conf | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/ldap/schema/slapd-collations.conf b/ldap/schema/slapd-collations.conf
index 9f653e4..31fa477 100644
--- a/ldap/schema/slapd-collations.conf
+++ b/ldap/schema/slapd-collations.conf
@@ -88,7 +88,7 @@ collation as "" "" 1 3	2.16.840.1.113730.3.3.2.71.1	as
 collation as IN "" 1 3	2.16.840.1.113730.3.3.2.72.1	as-IN
 collation az "" "" 1 3	2.16.840.1.113730.3.3.2.73.1	az
 collation az Latn "" 1 3	2.16.840.1.113730.3.3.2.74.1	az-Latn
-collation az Latn_AZ "" 1 3	2.16.840.1.113730.3.3.2.75.1	az-Latn-AZ
+collation az Latn AZ 1 3	2.16.840.1.113730.3.3.2.75.1	az-Latn-AZ
 collation bn "" "" 1 3	2.16.840.1.113730.3.3.2.76.1	bn
 collation bn BD "" 1 3	2.16.840.1.113730.3.3.2.77.1	bn-BD
 collation bn IN "" 1 3	2.16.840.1.113730.3.3.2.78.1	bn-IN
@@ -189,8 +189,8 @@ collation ha Latn NG 1 3	2.16.840.1.113730.3.3.2.172.1	ha-Latn-NG
 collation he "" "" 1 3	2.16.840.1.113730.3.3.2.173.1	he	he-IL
 collation hi "" "" 1 3	2.16.840.1.113730.3.3.2.174.1	hi	hi-IN
 collation hy "" "" 1 3	2.16.840.1.113730.3.3.2.175.1	hy	hy-AM
-collation id "" "" 1 3	2.16.840.1.113730.3.3.2.176.1	id-ID
-collation id ID "" 1 3	2.16.840.1.113730.3.3.2.177.1	ig-NG
+collation id "" "" 1 3	2.16.840.1.113730.3.3.2.176.1	id	id-ID
+collation ig "" "" 1 3	2.16.840.1.113730.3.3.2.177.1	ig	ig-NG
 collation it IT "" 1 3	2.16.840.1.113730.3.3.2.178.1	it-IT
 collation ka "" "" 1 3	2.16.840.1.113730.3.3.2.179.1	ka
 collation ka GE "" 1 3	2.16.840.1.113730.3.3.2.180.1	ka-GE
@@ -224,9 +224,9 @@ collation ru MD "" 1 3	2.16.840.1.113730.3.3.2.207.1	ru-MD
 collation ru RU "" 1 3	2.16.840.1.113730.3.3.2.208.1	ru-RU
 collation ru UA "" 1 3	2.16.840.1.113730.3.3.2.209.1	ru-UA
 collation si "" "" 1 3	2.16.840.1.113730.3.3.2.210.1	si	si-LK
-collation sk "" "" 1 3	2.16.840.1.113730.3.3.2.211.1	sk	sk-SK
-collation sl "" "" 1 3	2.16.840.1.113730.3.3.2.212.1	sl	sl-SI
-collation sq "" "" 1 3	2.16.840.1.113730.3.3.2.213.1	sq	sq-AL
+collation sk SK "" 1 3	2.16.840.1.113730.3.3.2.211.1	sk-SK
+collation sl SI "" 1 3	2.16.840.1.113730.3.3.2.212.1	sl-SI
+collation sq AL "" 1 3	2.16.840.1.113730.3.3.2.213.1	sq-AL
 collation sr Cyrl "" 1 3	2.16.840.1.113730.3.3.2.214.1	sr-Cyrl
 collation sr Cyrl BA 1 3	2.16.840.1.113730.3.3.2.215.1	sr-Cyrl-BA
 collation sr Cyrl ME 1 3	2.16.840.1.113730.3.3.2.216.1	sr-Cyrl-ME
-- 
2.4.11

